roles/dcos_bootstrap/tasks/main.yml

```diff
@@ -74,6 +74,54 @@
 - name: Create install directory/genconf
   file: path={{ download_path }}/genconf state=directory mode=0755
 
+- name: Create bootstrap CA directory
+  file: path={{ download_path }}/genconf/ca state=directory mode=0700
+
+- name: "Set PSK path"
+  set_fact:
+    psk_path: "{{ download_path }}/genconf/ca/psk"
+  when: dcos.enterprise_dcos
+
+- name: "Stat the existing PSK"
+  stat:
+    path: "{{ psk_path }}"
+  register: psk_stat
+  when: dcos.enterprise_dcos
+
+- name: "Read existing PSK contents"
+  slurp:
+    src: "{{ psk_path }}"
+  register: psk_contents
+  when: dcos.enterprise_dcos and psk_stat.stat.exists
+
+- name: "Generate PSK for exhibitor bootstrap"
+  shell: cat /dev/urandom | tr -dc 'a-z' | fold -w 16 | head -n1
```

Comment:
please add pipefail to this. The rest LGTM tests where successful

Reply:
👍